api/src/test/java/org/pmiops/workbench/api/DiskControllerTest.java

```diff
@@ -0,0 +1,331 @@
+    user.setUsername(LOGGED_IN_USER_EMAIL);
+    user.setUserId(123L);
+
+    createUser(OTHER_USER_EMAIL);
+
+    cdrVersion = new DbCdrVersion();
+    cdrVersion.setName("1");
+    // set the db name to be empty since test cases currently
+    // run in the workbench schema only.
+    cdrVersion.setCdrDbName("");
+    cdrVersion.setBigqueryDataset(BIGQUERY_DATASET);
+    DbWorkspace testWorkspace = new DbWorkspace();
+    testWorkspace.setWorkspaceNamespace(WORKSPACE_NS);
+    testWorkspace.setGoogleProject(GOOGLE_PROJECT_ID);
+    testWorkspace.setName(WORKSPACE_NAME);
+    testWorkspace.setFirecloudName(WORKSPACE_ID);
+    testWorkspace.setCdrVersion(cdrVersion);
+    doReturn(Optional.of(testWorkspace)).when(workspaceDao).getByNamespace(WORKSPACE_NS);
+  }
+
+  private static FirecloudWorkspace createFcWorkspace(
+      String ns, String googleProject, String name, String creator) {
+    return new FirecloudWorkspace()
+        .namespace(ns)
+        .name(name)
+        .createdBy(creator)
+        .googleProject(googleProject)
+        .bucketName(BUCKET_NAME);
+  }
+
+  private void stubGetWorkspace(
+      String workspaceNamespace, String googleProject, String firecloudName, String creator) {
+    DbWorkspace w = new DbWorkspace();
+    w.setWorkspaceNamespace(workspaceNamespace);
+    w.setFirecloudName(firecloudName);
+    w.setCdrVersion(cdrVersion);
+    w.setGoogleProject(googleProject);
+    when(workspaceDao.getRequired(workspaceNamespace, firecloudName)).thenReturn(w);
+    when(workspaceDao.getByNamespace(workspaceNamespace)).thenReturn(Optional.of(w));
+    stubGetFcWorkspace(
+        createFcWorkspace(workspaceNamespace, googleProject, firecloudName, creator));
+  }
+
+  private void stubGetFcWorkspace(FirecloudWorkspace fcWorkspace) {
+    FirecloudWorkspaceResponse fcResponse = new FirecloudWorkspaceResponse();
+    fcResponse.setWorkspace(fcWorkspace);
+    fcResponse.setAccessLevel(WorkspaceAccessLevel.OWNER.toString());
+    when(mockFireCloudService.getWorkspace(fcWorkspace.getNamespace(), fcWorkspace.getName()))
+        .thenReturn(fcResponse);
+  }
+
+  @Test
+  public void testGetPD() throws ApiException {
+    String readyPDName = user.generatePDName();
+    LeonardoListPersistentDiskResponse readyPDResponse =
+        new LeonardoListPersistentDiskResponse()
+            .name(readyPDName)
+            .size(300)
+            .diskType(LeonardoDiskType.STANDARD)
+            .status(LeonardoDiskStatus.READY)
+            .auditInfo(new LeonardoAuditInfo().createdDate("2021-08-06T17:57:29.827954Z"))
+            .googleProject(GOOGLE_PROJECT_ID);
+
+    LeonardoListPersistentDiskResponse deletingPDResponse =
+        new LeonardoListPersistentDiskResponse()
+            .name(user.generatePDName())
+            .size(300)
+            .diskType(LeonardoDiskType.STANDARD)
+            .status(LeonardoDiskStatus.DELETING)
+            .auditInfo(new LeonardoAuditInfo().createdDate("2021-08-06T16:57:29.827954Z"))
```

Comment:
you can add a test to make the DELETING's create date more recent, and see that happened